app/ui/parse_resume.py

```diff
@@ -1,82 +1,62 @@
 
 import spacy
 import gensim
-from gensim.corpora import Dictionary
+from gensim.corpora import Dictionary, MmCorpus
 from gensim.models.ldamodel import LdaModel
 from gensim.models import Phrases
 from gensim.models.phrases import Phraser
 from nltk.corpus import stopwords
+from spacy.matcher import Matcher
+from spacy.attrs import *
 import en_core_web_sm
 import re
 
+import codecs
+import pickle
+import os
+import warnings
+
 # yes it's a thing
 import nltk
 
 
 class SkillRecommender:
 
     def __init__(self,
-                 trigram_dictionary,
-                 bigram_model,
-                 trigram_model,
-                 lda_model,
-                 topic_names,
-                 hard_skills=[]):
+                 hard_skills,
+                 skills_dict,
+                 gensim_skills_dict,
+                 lda_model_final):
         """
         Create an instance of the thing we'll use to do skill
         parsing and recommendation
         """
 
         # check inputs
-        assert isinstance(trigram_dictionary, gensim.corpora.dictionary.Dictionary)
-        assert isinstance(bigram_model, gensim.models.phrases.Phrases)
-        assert isinstance(trigram_model, gensim.models.phrases.Phrases)
-        assert isinstance(lda_model, gensim.models.ldamodel.LdaModel)
-        assert isinstance(topic_names, dict)
+        # assert isinstance(trigram_dictionary, gensim.corpora.dictionary.Dictionary)
```

Comment:
I really hate leaving commented code behind. That makes things harder to read and edit later. We have git for a reason...just remove the lines we don't need anymore.

Also, you should add in new `assert` statements for the new stuff this method needs. I know we are on a time crunch, but those asserts are intended to reduce debugging time by allowing us to eliminate the possibility that a malformed object got passed in